core/templates/pages/exploration-editor-page/services/history-tab-backend-api.service.ts

```diff
@@ -24,11 +24,21 @@ interface HistoryTabBackendDict {
   'summaries': string[];
 }
 
-interface HistoryTabResposne {
+interface HistoryTabResponse {
   summaries: string[];
 }
 
-interface HistoryTabPostData{
+interface HistoryTabCheckRevertValidBackendDict {
+  'valid': boolean;
+  'details': string | null;
+}
+
+interface HistoryTabCheckRevertValidResponse {
+  valid: boolean;
+  details: string | null;
+}
```

Comment:
Aren't these basically the same?

Reply:
Ah yes -- I was following the pattern, but you're right they're identical. I'll clean those up.